test/seq_logger_tests.js

```diff
@@ -111,8 +111,21 @@ describe('SeqLogger', () => {
            assert.equal(sendBeacon.lastCall.args[1].size, 166);
        });
 
+       it('does handle event properties with circular structures', () => {
+        let logger = new SeqLogger({serverUrl: 'https://my-seq/prd', apiKey: '12345'});
+        const event = makeCircularTestEvent();
+        logger.emit(event);
+        try {
+            logger.flushToBeacon();
+        }
+        finally {
+            logger._wipeQueue();
+        }
```

Comment:
Just wondering what your thoughts are behind `_wipeQueue()`. The queue should be empty after line 119 if `flushToBeacon` succeeds. If `flushToBeacon` fails should we catch that error?

Reply:
wipeQueue Is not needed no, fixing in next commit :)